packages/quantic/cypress/fixtures/folded-result-list/exampleResultWithMoreChildResults.json

```diff
@@ -31,14 +31,15 @@
             "clickUri": "#4",
             "raw": {
               "foldingcollection": "Example Collection",
-              "foldingchild": ["parent2"],
-              "foldingparent": "parent1",
+              "foldingchild": ["node2"],
+              "foldingparent": "node1",
               "urihash": "#4"
             }
           },
           "raw": {
             "foldingcollection": "Example Collection",
-            "foldingparent": "parent2",
+            "foldingchild": ["leaf2"],
```

Comment:
@fbeaudoincoveo why do we need to add child to test results that didn't have any before?

Other than this, if the tests are green, go ahead :)

Reply:
It changes absolutely nothing on the test results, but I feel like it doesn't really make sense to have folded results that don't have a foldingchild value. Also, identifying the folded results with "child" / "leaf" makes it clearer what the intended hierarchy of the example result is IMO.

One possible drawback of this is that we no longer have test cases (at least as far as I'm aware) where the `getChildField` function in `folding-slice.ts` would return `undefined`.

I can revert this change if you prefer, it will have no other impact.